### ISSUE TITLE
New testers can't get work / No jobs are posted to Crowdflower

### ISSUE THREAD

**Issue by pscohn** [rainforestapp/tester-chrome-extension#24]
From JP:

> Many of us recently noticed that there are no more jobs posted on CrowdFlower platform any more for the past week. There are no JOB IDs and "Test a web application" task that appears in the CrowdFlower task window.

**Comment by ukd1** [comment]
Unless we've actually broken this code, it's likely a symptom of the work being "eaten" up by the chrome extension.

By eaten up, I mean it becomes visible to the chrome ext AND hit manager at the same time, the faster and more jobs get accepted by chrome, the less there are for the hitmanager to issue.

**Comment by pscohn** [comment]
That sounds probable since a lot of people have switched over to the chrome extension, though it raises an issue for people who have never tested for us before and want to start.

**Comment by frenks911** [comment]
Yea that felling when u get ban on the day the job id died #feelsbadman

**Comment by k65emil** [comment]
I suggest Rainforest to contact Crowdflower, because something _did_ happen at CF's end on Thursday November 19th, which caused "Test a web application" and most other jobs, but mainly RF and another one that only comes twice a week to stay "permanently" (though changing Job ID when new came) in CF, until Monday November 23rd. From that day there has been no sign of RF (nor the other job) in the  CF's job list (instead other jobs stays "permanently").

And it's a fact that many that used to get RF-hits via CF's job-list installed the extension to at least be able to get some RF-hits, which at the time of writing still are not showing up at all in CF's job-list.

**Comment by nberchev** [comment]
Yeah its the same problem for me. Recently got my first referal on CF and she is trying to reach your tests and really wants to work for you but without any submitted tasks she cant. So is there some decision for the people who had not worked for you until now? Can she make a profile on the RF portal itself and start working with the extension? Sry for bothering and I stay in anticipation of your answer. Thank you in advance. Regards, Nicolai

**Comment by ukd1** [comment]
@pscohn lets sync on this Monday.

**Comment by L4kshmi** [comment]
My account neobux was blocked by what they say use the pc to open multiple ads in one day ..! Now I can not work from there as RF Tester . I want to do it directly from Elite, but not how to install the application and that payments go to this account ... I would appreciate the help ..

**Comment by ptica-rugalica** [comment]
Any update about this? started to working on CF because of this task,and now when I finaly got my levels I can't. Are you leaving CF?

**Comment by L4kshmi** [comment]
As ptica-rugalica I also started in CF because I was commented on this work. There is some information? We can work in RF? Any comment about this?

**Comment by InvictusOne** [comment]
They have bigger "issues" right now... Like multiple users go in same terminal to do some task etc. And probably since holidays are here, guys that could handle that are most likely to drunk for that stuff :P hehehehehehe :D 

Jokes aside, perhaps this could be handled in the way like to have constant push to CF some "tutorial" tasks that would be free (meaning no one will get paid for those), like learning how to use terminal, report bugs, etc. And those tasks could have permanent ID's so if someone don't have linked IDs from some CF channel and TF they can create it and then use your chrome extension to do TF tasks. 

Kind regards and happy holidays :D

**Comment by L4kshmi** [comment]
Greetings! Any RF response about this?

**Comment by Josephapq** [comment]
Best regards.

Is there any progress in solving work in CrowdFlower? They have thought of a solution for new testers?

More than anything I wonder if at least in a short time the new testers (who have never worked in rainforest) may start using the extension.

Thank you.

**Comment by kaspan4o** [comment]
do you have any information on what happens to consumers who now want to start working with rainforestqa? 
do you think soon again to post tasks in crowdflower? Or you don't need new testers?

**Comment by jbarber** [comment]
@kaspan4o / @Josephapq / @L4kshmi we have a project in the works to solve this. I'm sure @pscohn will update you when it's available.

**Comment by goudi022** [comment]
i am not getting single job from last three weeks
my id is 30687845

**Comment by josejecr97** [comment]
is there any way to the new users join Rainforest? .... still no jobs are posted on CF

**Comment by goudi022** [comment]
I am not a new customer I am not getting hits after extension. I am member
since 2014

**Comment by ukd1** [comment]
@josejecr97 @goudi022 we're working on it - currently no, you have to wait for us to post - which takes a little while. We're going to re-work it soon.

**Comment by m2016cf** [comment]
I wait this job alot please open signup on rainforest only one day

**Comment by chmi16** [comment]
Is it possible to send ooops link to my referal and she just change user id (instead of mine  123456 she just put hers 654321)? Is this allowed? Will this be enough to get extension for new users?

**Comment by pscohn** [comment]
@chmi16 this is not allowed and she still won't be able to get paid by us if she isn't in our system.

**Comment by chmi16** [comment]
Ok. Thank you

**Comment by BonesKnows** [comment]
Do you guys have exact date when the job id will be posted on CrowdFlower?

**Comment by chandra95** [comment]
According to this issue almost all work is eaten by extension after launch(all most 3 months). I have few questions as a newbie
- If i refresh **CrowdFlower job listing**! is there any possibility to find out "**Test Web Application Job**"?. 
- **The system that posting of jobs** in CF listing is shutdown?
- Do you guys have **enough testers** and not interested in newbies?

**Comment by pscohn** [comment]
@chandra95 

- It is extremely unlikely
- It's not shut down, it's just taken by Chrome extension users before it can be posted to the CF listing. The fix for this involves pushing some new jobs to Crowdflower directly.
- We will always welcome and accept any new tester who wants to work for as long as it isn't to the detriment to the tester community as a whole.

**Comment by leka034** [comment]
I think that you want to avoid the answer here...Do you plan to push any job to Crowdflower directly or not? Is that such a problem?

**Comment by marius11** [comment]
@leka034 Jobs are sent to CrowdFlower, but they are immediately taken by those who use the Chrome extension.

**Comment by leka034** [comment]
@marius11 Yes, I understand that, but @pscohn said in the previous post that " The fix for this involves pushing some new jobs to Crowdflower directly", so I have asked him if it is possible?

**Comment by Drew-Goddyn** [comment]
In the meantime, could I suggest not posting a handful of jobs to crowdflower where they'll be 'eaten' by the extension, and reserve those to be provided manually/directly to new testers who go out of their way to inquire about beginning testing? Either that or allow direct access to the extension without needing to win the lottery on crowdflower till it resolved? 

As it stands, the very fact that this issue exists and has yet to be resolved after several months reflects very poorly for a company who is trying to SELL quality assurance as a product.

**Comment by leka034** [comment]
@pschon Can we expect to become a testers or that is nission impossible? Looking forward to your answer...

**Comment by pscohn** [comment]
@leka034 yes - it's still coming!

**Comment by jombere** [comment]
About how much time will be available on CD work? Dear time? gracais in advance

**Comment by leka034** [comment]
@pscohn Any news/updates about job posting on list? Or some other way to became testers?

**Comment by kristen-allie** [comment]
@leka034 Hello! I'm the new Ambassador to Testers for Rainforest. There are no updates, other than that it's still coming. There is no solid date yet, but I will update you all when there is one. I apologize for the delay! @jombere Does that answer your question as well?

**Comment by leka034** [comment]
@kristen-allie Thank you, we are waiting for you information...

**Comment by jombere** [comment]
Welcome! I hope we are very helpful and we keep abreast of all updates, very successful. @kristen-allie

**Comment by leka034** [comment]
@kristen-allie I wonder if all current testers quit testing for you, would you consider to do someting to acquire  new testers?

**Comment by BonesKnows** [comment]
@kristen-allie Is there any information?

**Comment by pscohn** [comment]
@BonesKnows Our core team is working on the new onboarding system right now - we don't have an exact date, but we estimate it should be ready in 3-4 weeks, maybe sooner :)

**Comment by L4kshmi** [comment]
@pscohn 4 months waiting for that :(. You always said that, it is very frustrating

**Comment by rojita7** [comment]
pscohn: Are through ptc? hopefully be direct with the rainforestqa
The ptc are taking many cents. Thank you.

**Comment by rojita7** [comment]
@pscohn I worry that reduce jobs by adding more users. 
Some have a rainforest as an important source of income

**Comment by L4kshmi** [comment]
@rojita7 All we want to work as TESTER can also take the job seriously. It sounds a bit selfish of you.

**Comment by ivevil** [comment]
I'm waiting 5 months to be a tester.. :( When can I become one? Is there any other ways to become one?

**Comment by marius11** [comment]
Read what @pscohn wrote a few lines up.

**Comment by jombere** [comment]
@pscohn  3 weeks have passed, what's next ?.

**Comment by Josephapq** [comment]
@jombere Three to four weeks friend. A little patience.

We are all desperate xD

**Comment by matheuskanderson** [comment]
@pscohn We have any news?

**Comment by ivijulsc** [comment]
Is it any way i can change my crowdflower ID to my RF account?
Today i registered, but i done the "Test a web aplication" a year ago, and my Previous NeoBux account got suspended for inactivity... So now i have the same CF account but a different NeoBux ID, and when i claimed my profile in RF i got linked with my previous NeoBux ID.
Any way to fix this? Thanks a lot.

**Comment by L4kshmi** [comment]
It would be easier to say they do not need more TESTER , that keep us on this one endless wait..! :(

**Comment by leka034** [comment]
@pscohn Do You have any news for us? Are you close to the final of that what are you working with?

**Comment by pscohn** [comment]
We're still working on it and getting closer, but I don't want to give any more vague estimates since it's taken longer than expected.. it's coming :)

**Comment by matheuskanderson** [comment]
@pscohn  But do you think that crowdflowers jobs return at least on The next month?

**Comment by ivijulsc** [comment]
@pscohn Well, that is good to know...
Where we shuld check for news? in this topic?

**Comment by marius11** [comment]
This topic or here: https://portal.rainforestqa.com/

**Comment by leka034** [comment]
@pscohn Is it coming? We are waiting? :(

**Comment by leka034** [comment]
@kristen-allie Can you at least give some “hope” ?...desperate to work with you...

**Comment by jeffreyvdhondel** [comment]
They said on the AMA at the end of this week. (skip to 26:50)
[Youtube AMA](https://youtu.be/igBS6_csYNw?t=26m50s)

**Comment by ilegalna** [comment]
Greetings!

When new testers arrive, is that mean less work for everyone and more problems with VM, much more messages like "There are no virtual machines currently available. Please retry in a few minutes" etc.? 

Did you reduce the number of old testers and, if so, will you introduce stricter rules? 

Thanks for your reply. I believe that the testers should not suffer if newcomers get a chance.

**Comment by mariaelite** [comment]
Ilegalna: completely agree with you

**Comment by mariaelite** [comment]
@L4kshmi Sorry but it's true

**Comment by marius11** [comment]
@mariaelite Where did you get that information? There are testers have have done these tasks for the past 18 months. Why would anyone "replace" them with new testers?

**Comment by mariaelite** [comment]
@marius11 Read forum

**Comment by marius11** [comment]
@mariaelite Could you be more specific please? I have read Rainforest's AMA (you should too) and it answers a lot of questions, including adding more testers. They said that they are having new customers every week or so, and the need for more testers makes perfect sense in this case.

**Comment by mariaelite** [comment]
@marius11 Simply experience.
We live last year.
The only people who can give guarantee otherwise is Raiforest.
Until then, it follows the concern.

**Comment by ivijulsc** [comment]
Any news about new testers? We are waiting :)
Thanks!

**Comment by leka034** [comment]
Nice, you said that the job, TW, will be on the list last week or at the begining of this week...So??

**Comment by jeffreyvdhondel** [comment]
Again promises but not keeping them.....

**Comment by braneb** [comment]
there is barely work for current testers...you will get your chance when you get it so please stop whining

**Comment by marius11** [comment]
@jeffreyvdhondel Nobody promised anything, except that tests will be available to new testers in the near future. They have their good reasons not to give a date.

**Comment by NeoWorker** [comment]
Yeah, I'm getting tired of this. But I still love ya Rainforest <3

**Comment by leka034** [comment]
We all love you :)

**Comment by matheuskanderson** [comment]
is today

**Comment by jombere** [comment]
@matheuskanderson  Today they are going to launch the job to the list of CW?

**Comment by leka034** [comment]
@kristen-allie Any news about new testers?

**Comment by leka034** [comment]
pscohn Any news about new testers?

**Comment by leka034** [comment]
@pscohn Any news about new testers? I think that @kristen-allie is angry on me :(

**Comment by leka034** [comment]
@kristen-allie @pscohn I have a question, if you are able to provide me with an answer. Why do you send training tasks trough extension to testers who already worked for you? And than send them a congratulation msg for becoming a tester which they already are?

**Comment by vvarna** [comment]
I'm NOT new tester. I have my account created doing tests via CF task in last 1 year provided in Neobux. Last year all was normal and all the commissions was paid via my Neobux channel. I'm also seaman by the profession and till May this year I was not at home. From May till now I worth around 115$ using only Chrome extension and all the money just pending in my existing Rainforest account. It's like no way to take them because to be paid I need to complete a one task at least via CrowdFlower. But they haven't how I read from last year. Anyway I will continue here because I believe Rainforest. So guys just be patient. We are just part (important part) from something bigger.

**Comment by farisxd** [comment]
There was a job posted on CF Task list but it gone in just few minutes, why? Why can all we get a chance?

**Comment by matheuskanderson** [comment]
@farisxd it is true ? you did?

**Comment by rftester2016** [comment]
Yes it is true.It was active for only couple of minutes(3-4) named with "Test Applications For Rainforest Qa" or something like that(forgot the exact name).

**Comment by svtldav** [comment]
Did anyone remember the exact name of the task in the list?

**Comment by marinova** [comment]
vvarna i am in a same situation...i have penning around 500$ and waiting for the new number to get the payment key ...

**Comment by begsoon** [comment]
@farisxd  Can you be more realistic please :) There is maybe 5 000 - 10 000 new people that want to also to get test in Rainforest,do you think that right now is possible for rainforest to get that number of new testers..
You should keep eye on tasks screen of CF

**Comment by saulruiz** [comment]
I can do work normally rainforest , and then when I make at least one in CrowdFlower , they paid me all I did before? sorry for bad english...

**Comment by matheuskanderson** [comment]
she just popped up in CrowdFlower list

**Comment by vvarna** [comment]
Hello saulruiz,

Once when you complete one task at least via CrowdFlower you will get paid. At this moment Rainforest QA together with CrowdFlower try to deliver new task but CrowdFlower fail it. But don't worry soon you will get it and will be able to take your money.

**Comment by matheuskanderson** [comment]
![rain](https://cloud.githubusercontent.com/assets/18285858/15979900/66f47026-2f3d-11e6-9d05-b455db9ed6b5.png)

**Comment by rftester2016** [comment]
So yesterday and today rainforest does post a job in CF job page.But the problem is you should be super lucky to catch it.It stays only for minutes if not seconds.

**Comment by vvarna** [comment]
So guys and girls. In the past one hour the Task was available for limited time again via CrowdFlower channels.

**Comment by leka034** [comment]
Question for someone from RF. I manage do catch task on the CF list last night and it is in my history of comlpeted jobs, but I havent received any notification with link for chrome extension? I olso wrote support ticket but there is no reply from your support team.

**Comment by luiscarreno74** [comment]
the task came in crowdflower but was kind of wroken

**Comment by cobijin** [comment]
I have done the CF task today, registered for very first time, installed the chrome ext. and done the training mode in the Rainforest Terminal. Everything was ok.

**Comment by matheuskanderson** [comment]
here are not paying, I've done cf task, does anyone know why?

**Comment by luiscarreno74** [comment]
The task that i did was just a box to click when read instructions then submit and nothing more and no email was send to me

**Comment by ivijulsc** [comment]
What time is the Job posted?
I was refreshing NeoBux evry 10 minutes but never see it :(

**Comment by vrr123** [comment]
I confirmed the task on the CF list. But I have not received any notification. Support does not answer yet. Who has the same problems?  I'm desperate...

**Comment by luiscarreno74** [comment]
I have the same problem vrr123 i dont know wjat to do coz un instruction says that if se do twice de will be banned bit thats an error of the task

**Comment by ayubanget** [comment]
@luiscarreno74: Have the same problem. I also only see a checkbox and no link to any task.

**Comment by Vasil-RainforestQA-tstrs** [comment]
@luiscarreno74 , @ayubanget 

Please visit the next URL and try to find notification from Rainforest QA .regarding your profile.

https://tasks.crowdflower.com/notifications

**Comment by pscohn** [comment]
As of late last week we are posting a small number of jobs to CrowdFlower each day. To keep the intake of new testers manageable we're limiting the number that can get a task on a daily basis, so it might take some luck at first.

If you don't receive a notification or email about claiming a profile, let us know by emailing testers@rainforestqa.com with your worker ID and any relevant screenshots.

**Comment by luiscarreno74** [comment]
Hi @Vasil-RainforestQA-tstrs @pscohn i didn't received a notification or email about claiming my profile, i send an email to that direction 3 days ago but didnt get any responses, i send an screen shot and my worker id on 10/06/2016, please help i want to be rain tester too

**Comment by pscohn** [comment]
@luiscarreno74 I have replied to your ticket.

**Comment by ayubanget** [comment]
@pscohn @Vasil-RainforestQA-tstrs Same as @luiscarreno74, i also didn't received any notification or email. I also have send a ticket to testers@rainforestqa.com as @pscohn instructed just know. Can someone help?

Thanks for your help.

**Comment by luiscarreno74** [comment]
thanks @pscohn i noticed that i didnt send you the screenshot with my notifications in crowdflower so i reply the ticket, because i didnt get ant notification from the task. thanks in avanced

**Comment by vrr123** [comment]
@pscohn  I also didn't received any notification. Can you help me, too, please? I wrote in support on 12 June. id and a screen shot  attached to the letter. last digits of my email - 468@gmail Thank you.

**Comment by matheuskanderson** [comment]
@pscohn I have sent e- mail too but no one answers , can you help me ? Thank you!

**Comment by pscohn** [comment]
We're looking into the issue with notifications, will post when we have an update.

**Comment by ayubanget** [comment]
Hi @pscohn any update about the notifications issue? I'm still doesn't received any notification.

Thanks for your help.

**Comment by mahonikv** [comment]
Hi https://github.com/pscohn I have generated name Rainforest but when i want to claim arrive message "You will receive a message soon with further instructions." but 4 day I there not instruction.

**Comment by ivijulsc** [comment]
@mahonikv i recibed the message few minutes after i claim it in my "CF" account.

**Comment by mahonikv** [comment]
ivijulsc I wait 4 dey yet.

**Comment by larissasilva** [comment]
@pscohn there's something wrong with my bonus system I'm not getting my payments already completed the task CrowdFlower

**Comment by OanceaDenis97** [comment]
Can you post the jobs earlier? Some people who are from the USA may not be able to access them.

**Comment by OanceaDenis97** [comment]
who are not*

**Comment by NeoWorker** [comment]
What is a double-submission of task?

**Comment by jeloy1** [comment]
I have two questions. What is the job of the task ID? and What is the name of the task? I'm confused. Please help.

**Comment by Vasil-RainforestQA-tstrs** [comment]
@jeloy1 
Regarding your first question. The Rainforest QA tasks  provided via CrowdFlower and
Amazon Turk channels is named / or similar  “Test  Applications for Rainforest Qa”. 

To avoid asking for already answered questions please read the FAQ page here: https://github.com/rainforestapp/tester-chrome-extension/wiki/Frequently-Asked-Questions

**Comment by jeloy1** [comment]
I've waited long enough task, but nothing appears to me, on any platform, or gifthunter, clixsemse or cro elite. Anyone have any idea more or less what time does the task?

**Comment by Vasil-RainforestQA-Tester** [comment]
So guys. Exactly now the task is available via Crowdflower channel. Enjoy it !

**Comment by mahonikv** [comment]
Hi @Vasil I have generated name Rainforest but when i want to claim arrive message "You will receive a message soon with further instructions." but 10 day I there not instruction.

**Comment by Vasil-RainforestQA-Tester** [comment]
@mahonikv Hi dear. As tester I understand the importance of your issue. 
The right way is to address your ticket to:  tester@rainforestqa.com  
Don`t forget to attach screenshots (if available) or other important information like Worker ID etc.

**Comment by jeloy1** [comment]
I was working on crowflovers tasks from gifthunterclub and closed that platform, then I opened my account croflover elite and did some work, but forget to change my ID worker gifthunter at croflower elite, then took the task of rainforesqa from crowflover elite and reviewed notifications I followed the instructions, but to verify my story in the profile rainforesqa left me with worker id gifthunter and not crowflover elite will there be problems when collecting rain hit gifthunterclub from my account? another question is, when I opened my profile rain, I could see that I get another name, and down the name of this written "This is a generated name, claim your profile to Set your name." I myself change it to a name or leave that? Please forgive many messages. I'm curious and I like to be aware of the small details, I appreciate your help.

**Comment by larissasilva** [comment]
@Vasil-RainforestQA-Tester there is something wrong with my bonus system that I 'm not getting my payments already completed the task CrowdFlower already sent email is 8 days and no one answers me can you please help me?

**Comment by Vasil-RainforestQA-Tester** [comment]
@larissasilva Hallo. You need to address your question to the respective staff member or just send ticket from your account. My advice is first to check your Rainforest account for the related bonus transaction. If Rainforest mark as paid your bonus you need to contact CrowdFlower for this issue because this is beyond of their control.

**Comment by larissasilva** [comment]
@Vasil-RainforestQA-Tester Hello , my account this way. I was already RF before opening the CrowdFlower.
![image](https://cloud.githubusercontent.com/assets/19961019/16207616/69b20d0e-3704-11e6-9870-0cbb3e11c9c8.png)

**Comment by Vasil-RainforestQA-Tester** [comment]
To take your collected funds first you need to complete one task at least provided via CrowdFlower channel. Once the task is completed you will be notified that your payment information is updated. This is the only way man. Just refresh the task list and wait new task to be released.

**Comment by larissasilva** [comment]
@Vasil-RainforestQA-Tester I've completed the task at CrowdFlower , and yet I'm still not getting paid ..

**Comment by jeloy1** [comment]
I was working on crowflovers tasks from gifthunterclub and closed that platform, then I opened my account croflover elite and did some work, but forget to change my ID worker gifthunter at croflower elite, then took the task of rainforesqa from crowflover elite and reviewed notifications I followed the instructions, but to verify my story in the profile rainforesqa left me with worker id gifthunter and not crowflover elite will there be problems when collecting rain hit gifthunterclub from my account? another question is, when I opened my profile rain, I could see that I get another name, and down the name of this written "This is a generated name, claim your profile to Set your name." I myself change it to a name or leave that? Please forgive many messages. I'm curious and I like to be aware of the small details, I appreciate your help.

**Comment by angelfelixgarciamolina** [comment]
The problem of the ID's is de CrowFlowder o RainForest? The paid is aboned in RF but not arrived in CF

**Comment by jeloy1** [comment]
I did training in rainforestqa and I passed successfully, I got the notice: "Your training is complete Hi guerraeloy1, Great news: your training is complete You will now be reliable to do regularly work for us Happy testing Rainforest QA!!! Team ps view your profile here ... " but is not reflected bonds each task on my profile rainforestqa. Nor is it reflected in my ClixSense.

**Comment by jeloy1** [comment]
I did training in rainforestqa and I passed successfully, I got the notice:
"Your training is complete Hi guerraeloy1, Great news: your training is
complete You will now be reliable to do regularly work for us Happy testing
Rainforest QA!!! Team ps view your profile here ... "but is not reflected
bonds each task on my profile rainforestqa. Nor is it reflected in my
ClixSense.




2016-06-23 2:56 GMT-04:00 Angel Felix Garcia Molina <
notifications@github.com>:

**Comment by Vasil-RainforestQA-Tester** [comment]
Guys all we experience almost same problems. At that moment as regular tester I can confirm that Rainforest try to respond quickly as posible of any issue and support tickets feom all of us.  We are so many ! Try to think from their side too. Its not so easy to be managed and separately our payments depends from third party - CrowdFlower and/or their contactors. Be confident. Most of the issues depends from CrowdFlower and the right wall to be placed is their Support center who you will find on the upper right corner of your CrowdFlower account. 
Some of you already blocked CrowdFlower accounts. This also mean that your payment channel is disconnected by CrowdFlower and you cannot get your money from Rainforest Test. Do not place your tickets to RainForest. The only way to get your CF account back is to write to CrowdFlower Support center.
At that moment I would like to thanks Rainforest and all of you for hard work . And keep clicking - the tests comming !

**Comment by maciejgryka** [comment]
@pscohn Hope it's OK to close this? We now regularly post training jobs for new people on CF.

**Comment by rezkymea** [comment]
i can work on rainforestqa again on new channel, there is new update for people who lose the old channel can change it to new one

**Comment by dannyvaya** [comment]
hi, I'm a new member.
How can I join rainforest qa for now? 
Thank you in advance! much appreciate.

**Comment by sweeneyapps** [comment]
@dannyvaya 

sorry, as of right now, RainforestQA is not accepting any new testers.
